transports/janus_websockets.c

```diff
@@ -843,18 +847,22 @@ void janus_websockets_session_created(janus_transport_session *transport, guint6
 }
 
 void janus_websockets_session_over(janus_transport_session *transport, guint64 session_id, gboolean timeout) {
-	if(!timeout || transport == NULL || transport->transport_p == NULL || g_atomic_int_get(&transport->destroyed))
+	if(!timeout || transport == NULL || g_atomic_int_get(&transport->destroyed))
```

Comment:
Are we always sure that `transport_p` can be always trusted to be valid? I'd keep the check just for safety...